src/Infrastructure/Locale/LocaleSubscriber.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright 2010 SURFnet B.V.
```

Comment:
Please set this to either the year this product was written, or use the current year. Also applies for the associated test

Reply:
I'll  change this all at once in https://github.com/OpenConext/Stepup-ReadID/pull/27